beacon_node/execution_layer/src/lib.rs

```diff
@@ -1269,3 +1318,11 @@ mod test {
 fn noop<T: EthSpec>(_: &ExecutionLayer<T>, _: &ExecutionPayload<T>) -> Option<ExecutionPayload<T>> {
     None
 }
+
+/// Returns the duration since the unix epoch.
+pub fn timestamp_now() -> u64 {
```

Comment:
Can we move this function under the `cfg(test)` so it doesn't accidentally get used for something else?